tests/commands/update/test_update_panelapp_green.py

```diff
@@ -6,12 +6,12 @@
 
 from scout.commands import cli
 from scout.demo import panelapp_panel_path, panelapp_panels_reduced_path
-from scout.load.panelapp import PANEL_NAME
 from scout.server.extensions import store
 from scout.server.extensions.panelapp_extension import API_PANELS_URL
 
 DEMO_PANEL = "522"
 PANELAPP_GET_PANEL_URL = f"{API_PANELS_URL}{DEMO_PANEL}"
+PANEL_NAME = "PANELAPP-GREEN"
```

Comment:
Perhaps move this constant to /scout/constants/panels and name it "PANELAPP_GREEN_PANEL_ID". Then it can be imported in the CLI and in here.